app/models/metric/purging.rb

```diff
@@ -50,49 +50,38 @@ def self.purge_timer(ts, interval)
     end
   end
 
-  def self.purge_count(older_than, interval)
-    klass, conditions = case interval
-                        when 'realtime' then [Metric,       {}]
-                        else             [MetricRollup, {:capture_interval_name => interval}]
-                        end
-
-    oldest = klass.select(:timestamp).where(conditions).order(:timestamp).first
-    oldest = oldest.nil? ? older_than : oldest.timestamp
+  def self.klass_for_interval(interval)
```

Comment:
lol. well, it used to return a class until you asked me to change it.

you have a better name?